src/protojure/grpc/client/utils.clj

```diff
@@ -31,7 +33,9 @@ after the channel is closed.  Used in remote procedure calls with unary paramete
 | **params**  | _any_                | The object to place on the channel                                         |
   "
   [ch params]
-  (-> (put ch params)
+  (-> (if (some? params)
```

Comment:
I think you can just drop this whole hunk.  It would be covered by the changes you made to L19-L21.